manifests/w/WestermannGmbH&CoKG/BiBox2/0/2.3.5/WestermannGmbH&CoKG.BiBox2.0.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: WestermannGmbH&CoKG.BiBox2.0
+PackageVersion: 2.3.5
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://static.bibox2.westermann.de/apps/windows

```

```suggestion
  InstallerUrl: https://static.bibox2.westermann.de/electron/autoUpdate/BiBox%202.0%202.3.5%20Setup.exe
```
Versioned-URL preferred.